debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectionFactory.java

```diff
@@ -5,32 +5,34 @@
  */
 package io.debezium.connector.oracle;
 
-import static io.debezium.connector.oracle.OracleConnectorConfig.CONNECTOR_ADAPTER;
-
 import java.sql.Connection;
-import java.sql.DriverManager;
 import java.sql.SQLException;
 
 import io.debezium.connector.oracle.OracleConnectorConfig.ConnectorAdapter;
 import io.debezium.jdbc.JdbcConfiguration;
+import io.debezium.jdbc.JdbcConnection;
 import io.debezium.jdbc.JdbcConnection.ConnectionFactory;
 
 public class OracleConnectionFactory implements ConnectionFactory {
 
+    private static final String DRIVER_TYPE_THIN = "thin";
+    private static final String CONNECTION_ADAPTER = "connection.adapter";
+    private static final String USE_FETCH_SIZE_WITH_LONG_COLUMN = "useFetchSizeWithLongColumn";
+
     @Override
     public Connection connect(JdbcConfiguration config) throws SQLException {
-        String hostName = config.getHostname();
-        int port = config.getPort();
-        String database = config.getDatabase();
-        String user = config.getUser();
-        String password = config.getPassword();
-
-        final String url = "jdbc:oracle:" + getDriverType(config) + ":@" + hostName + ":" + port + "/" + database;
-
-        return DriverManager.getConnection(url, user, password);
+        final String driverType = getDriverType(config);
+        if (DRIVER_TYPE_THIN.equals(driverType)) {
+            System.out.println("Configs: " + config.keys());
```

Comment:
@Naros Hi, I think this code was not run through formatter + checkstyle.

Reply:
LOL, thanks.  I'll fix & rebase with master since you merged the other PR.

Reply:
Fixed